src/Synth/LFO.h

```diff
@@ -18,8 +18,18 @@
 #include "../Misc/Time.h"
 #include "WatchPoint.h"
 
+
+
 namespace zyn {
 
+typedef enum lfo_state_type{
```

Comment:
Consider putting this as `private` into the class.

Reply:
ok